tests/cases/fourslash/codeFixConvertToTypeOnlyImport4.ts

```diff
@@ -10,13 +10,14 @@
 ////import { I, foo } from "./b";
 
 goTo.file("/a.ts");
+
+verify.codeFixAvailable([
+    { description: "Use 'type I'" },
+    { description: "Remove import from './b'" }
+]);
+
 verify.codeFix({
     index: 0,
-    description: ts.Diagnostics.Use_import_type.message,
-    newFileContent: `import type { I, foo } from "./b";`
```

Comment:
This is a valid fix and it’s intended to be offered, because `foo` is never used in value space in `a.ts`. I think the right fix is to recognize the non-type-only re-export in the new test case as a value-space usage in the FindAllReferences API call.

Reply:
Oke, thanks